modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/webdav/DLWebDAVStorageImpl.java

```diff
@@ -1220,6 +1220,19 @@ protected Resource toResource(
 		return resource;
 	}
 
+	private FileEntry _getFileEntryByName(
+			long groupId, long parentFolderId, String name)
+		throws PortalException {
+
+		try {
+			return _dlAppService.getFileEntry(groupId, parentFolderId, name);
+		}
+		catch (NoSuchFileEntryException noSuchFileEntryException) {
+			return (FileEntry)DLFileEntryUtil.findByG_F_FN(
```

Comment:
Is there any reason you're calling the persistence directly? Here we should be calling DLFileEntry*Service methods only.

Reply:
Nevermind, I see you change this later.

Reply:
There is no reason.
In this point this is totally wrong.
Thanks